playback/Cargo.toml

```diff
@@ -91,6 +91,7 @@ symphonia = { version = "0.5", default-features = false, features = [
     "mp3",
     "ogg",
     "vorbis",
+    "flac"
```

Comment:
Although not useful for Spotify Lossless until we find a way to get the decryption keys, it _is_ useful for local file support in #1595 so I'm gonna merge it anyway.